<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>1021</width>
      <height>701</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string/>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,4">
     <item>
      <widget class="QFrame" name="horizontalFrame">
       <property name="styleSheet">
        <string/>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
        <item>
         <widget class="QFrame" name="verticalFrame_4">
          <property name="styleSheet">
           <string/>
          </property>
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="ipName">
               <property name="styleSheet">
                <string/>
               </property>
               <property name="text">
                <string>IP_Device</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="edIPDevice">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>35</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>35</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QLineEdit {
    border-top: 1px solid gray;
    border-right: 1px solid gray;
    border-left: 1px solid gray;
    border-bottom: 1px solid gray;
}</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="styleSheet">
                <string/>
               </property>
               <property name="text">
                <string>Device_Name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="edDevice">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>35</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>35</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QLineEdit {
    border-top: 1px solid gray;
    border-right: 1px solid gray;
    border-left: 1px solid gray;
    border-bottom: 1px solid gray;
}</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="checkConnect">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>35</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
    background-color: #4CAF50;  /* Màu nền */
    color: white;                /* Màu chữ */
    font: bold 14px Arial;       /* Font chữ, kiểu chữ, kích thước */
    border: 2px solid #4CAF50;  /* Viền nút */
    border-radius: 10px;         /* Bo góc nút */
    padding: 10px 20px;          /* Khoảng cách nội dung trong nút */
}

QPushButton:hover {
    background-color: #45a049;  /* Màu nền khi di chuột vào */
}

QPushButton:pressed {
    background-color: #387c44;  /* Màu nền khi nhấn nút */
}</string>
               </property>
               <property name="text">
                <string>Check Connect</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,5">
             <item>
              <widget class="QCheckBox" name="checkAll">
               <property name="text">
                <string>CheckAll</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="sendAll">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>35</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>250</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
    background-color: #4CAF50;  /* Màu nền */
    color: white;                /* Màu chữ */
    font: bold 14px Arial;       /* Font chữ, kiểu chữ, kích thước */
    border: 2px solid #4CAF50;  /* Viền nút */
    border-radius: 10px;         /* Bo góc nút */
    padding: 10px 20px;          /* Khoảng cách nội dung trong nút */
}

QPushButton:hover {
    background-color: #45a049;  /* Màu nền khi di chuột vào */
}

QPushButton:pressed {
    background-color: #387c44;  /* Màu nền khi nhấn nút */
}</string>
               </property>
               <property name="text">
                <string>Send All Ccommand</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1">
          <item>
           <widget class="QFrame" name="verticalFrame_3">
            <property name="styleSheet">
             <string/>
            </property>
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QTextBrowser" name="logshow"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="5,1">
       <item>
        <widget class="QFrame" name="verticalFrame_3">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>991</width>
               <height>428</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="horizontalFrame_2">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    border-bottom: none;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="singleCmd">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
    border-top: 1px solid gray;
    border-right: 1px solid gray;
    border-left: 1px solid gray;
    border-bottom: 1px solid gray;
}</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="send">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>50</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Send</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1052</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
